docs/infra/website.md

```diff
@@ -0,0 +1,196 @@
+Website (fossrit.github.io)
+===========================
+
+This page documents the official FOSS@MAGIC website hosted on GitHub Pages.
+The website source code is found in [FOSSRIT/fossrit.github.io](https://github.com/FOSSRIT/fossrit.github.io "Official website for Free and Open Source Software @ RIT MAGIC Center and FOSS academia").
+
+
+## About
```

Comment:
Given how you're structuring docs across `README.md`, `CONTRIBUTING.md`, and here, it may be worth putting this to `CONTRIBUTING.md`

Reply:
I'm keeping track of this in FOSSRIT/fossrit.github.io#86. :+1: